climada/hazard/tc_clim_change.py

```diff
@@ -1,174 +1,375 @@
+        2.644714286,2.688414286,2.688514286,2.685314286,2.724614286,2.746214286,2.773814286
+    ]
 
+    gmst_info['gmst_data'][3] = [
+        -0.16,-0.08,-0.1,-0.16,-0.28,-0.32,-0.3,-0.35,-0.16,-0.1,-0.35,-0.22,-0.27,
+        -0.31,-0.3,-0.22,-0.11,-0.11,-0.26,-0.17,-0.08,-0.15,-0.28,-0.37,-0.47,-0.26,
+        -0.22,-0.39,-0.43,-0.48,-0.43,-0.44,-0.36,-0.34,-0.15,-0.14,-0.36,-0.46,-0.29,
+        -0.27,-0.27,-0.19,-0.28,-0.26,-0.27,-0.22,-0.1,-0.22,-0.2,-0.36,-0.16,-0.1,-0.16,
+        -0.29,-0.13,-0.2,-0.15,-0.03,-0.01,-0.02,0.13,0.19,0.07,0.09,0.2,0.09,-0.07,-0.03,
+        -0.11,-0.11,-0.17,-0.07,0.01,0.08,-0.13,-0.14,-0.19,0.05,0.06,0.03,-0.02,0.06,0.04,
+        0.05,-0.2,-0.11,-0.06,-0.02,-0.08,0.05,0.02,-0.08,0.01,0.16,-0.07,-0.01,-0.1,0.18,
+        0.07,0.16,0.26,0.32,0.14,0.31,0.15,0.11,0.18,0.32,0.38,0.27,0.45,0.4,0.22,0.23,0.32,
+        0.45,0.33,0.47,0.61,0.39,0.39,0.54,0.63,0.62,0.54,0.68,0.64,0.66,0.54,0.66,0.72,0.61,
+        0.64, 0.68,0.75,0.9,1.02,0.92,0.85,0.98,0.945764286,1.011064286,1.048564286,1.049564286,
+        1.070264286,1.126564286,1.195464286,1.215064286,1.246964286,1.272564286,1.262464286,
+        1.293464286,1.340864286,1.391164286,1.428764286,1.452564286,1.494164286,1.520664286,
+        1.557164286,1.633664286,1.654264286,1.693264286,1.730264286,1.795264286,1.824264286,
+        1.823864286,1.880664286,1.952864286,1.991764286,1.994764286,2.085764286,2.105764286,
+        2.155064286,2.227464286,2.249964286,2.313664286,2.341464286,2.394064286,2.457364286,
+        2.484664286,2.549564286,2.605964286,2.656864286,2.707364286,2.742964286,2.789764286,
+        2.847664286,2.903564286,2.925064286,2.962864286,3.002664286,3.069264286,3.133364286,
+        3.174764286,3.217764286,3.256564286,3.306864286,3.375464286,3.420264286,3.476464286,
+        3.493864286,3.552964286,3.592364286,3.630664286,3.672464286,3.734364286,3.789764286,
+        3.838164286,3.882264286,3.936064286,3.984064286,4.055764286,4.098964286,4.122364286,
+        4.172064286,4.225264286,4.275064286,4.339064286,4.375864286,4.408064286,4.477764286
+        ]
+
+    return gmst_info
 
-def calc_scale_knutson(ref_year=2050, rcp_scenario=45):
+def get_knutson_data():
     """
-    Comparison 2081-2100 (i.e., late twenty-first century) and 2001-20
-    (i.e., present day). Late twenty-first century effects on intensity and
-    frequency per Saffir-Simpson-category and ocean basin is scaled to target
-    year and target RCP proportional to total radiative forcing of the
-    respective RCP and year.
+    Get data to generate figures 1b to 4b in Knutson et al., (2020):
 
-    Parameters
-    ----------
-    ref_year : int, optional
-        year between 2000 ad 2100. Default: 2050
-    rcp_scenario: int, optional
-        26 for RCP 2.6, 45 for RCP 4.5. The default is 45
-        60 for RCP 6.0 and 85 for RCP 8.5.
+    Tropical cyclones and climate change assessment. Part II: Projected 
```

Comment:
Great to add the sources! Could you please clean-up the formatting?

Reply:
Please resolve all `trailing-whitespace` issues encountered by Jenkins. You see the respective annotations in the "Changes" tab of the PR